### PR TITLE
Turn into a Stencil project

### PR DESCRIPTION
**Pull request by ThibodeauJF**

### CODE REVIEW

**Review by olamothe — COMMENTED** [review]
It seems pretty good overall. 

The

**Comment by btaillon** [comment]
Since, as far as I understand, the goal of creating this kind of project with the CI is to create custom components, not a search page, I think we should:
1. Allow somebody to specify a search token instead of an API token + email.
2. Use the sample org by default instead of requiring configuration
    * Perhaps we could even list some of the fields available in the org in the README (or link to some doc with the information)

Additionally, I think we should add in the README that people should execute `npm install` before doing any other `npm` command. I don't know if everybody wanting to make a custom component is necessarily gonna be familiar with npm. 

Currently, I can't compile the project, I'm getting errors such as the following:
image

Not sure what's wrong.

**Review by btaillon — APPROVED** [review]

File: src/components/sample-component/sample-component.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+export class SampleComponent implements AtomicComponent {
+  pagerController!: Pager;
+  statusController!: SearchStatus;
+
+  @BindStateToController("pagerController") @State() pagerState!: PagerState;
+  @BindStateToController("statusController")
+  @State()
+  statusState!: SearchStatusState;
+
+  connectedCallback() {}
```
[the remark this answers is not in the document]
Comment (ThibodeauJF) [reply src/components/sample-component/sample-component.tsx:31]: You need to have them, I still couldn't figure why in this project vs the ui-kit one it worked without

**Comment by ThibodeauJF** [comment]
@btaillon I npm linked my local atomic version to get the most recent changes, I think it works if you get the alpha instead of the latest version too.

For all the rest of the setup, this is an example of an atomic-cli generated project where everything is set up for you, installed packages, api keys and etc

**Pull request closed without merging**